frappe/database/mariadb/setup_db.py

```diff
@@ -71,8 +66,8 @@ def bootstrap_database(verbose, source_sql=None):
 	import sys
 
 	frappe.connect()
-	if not check_database_settings():
-		print("Database settings do not match expected values; stopping database setup.")
+	if not check_compatible_versions():
+		print("MariaDB version compatibility checks failed, make sure you're running a supported version; stopping database setup.")
 		sys.exit(1)
```

Comment:
```suggestion
	check_compatible_versions()
```
Leaving it like this would be equivalent to current behavior. In the proposed change, the function would always return None and cause this to exit?